### DISCUSSION TITLE
Blazored SessionStorage vs ProtectedBrowserStorage

### DISCUSSION THREAD

**Discussion by joe912** [Blazored/SessionStorage#80] [Q&A] [answered]
Hello,
Microsoft.AspNetCore.Components.Server.ProtectedBrowserStorage provides session storage ([link](https://learn.microsoft.com/en-us/aspnet/core/blazor/state-management?view=aspnetcore-7.0&pivots=server))

Is there a difference between the this package and ProtectedBrowserStorage?  Or was the package developed to provide the functionality now found in ProtectedBrowserStorage?

Thanks 
Joe

**Comment by chrissainty** [comment, marked as answer]
This package was developed long before the Microsoft package was developed. The main difference is that their package offers some obfuscation of data, but this one does not. However, I think we have more features overall plus integration for testing with bUnit

**Reply by joe912** [reply]
Thanks for reply Chris.  I appreciate your work for the Blazor community - keep up the good work :)
